pouchdb-core/pouchdb-core.d.ts

```diff
@@ -245,63 +537,177 @@ declare namespace PouchDB {
                 RemoteDatabaseConfiguration;
     }
 
+    /** @todo: remove nodejs dep. */
+    interface EventEmitter extends NodeJS.EventEmitter {
 
+    }
 
-    interface Static {
+    interface Static extends EventEmitter {
```

Comment:
@nolanlawson Is PouchDB an `EventEmitter` outside of Node?  ([api#events](https://pouchdb.com/api.html#events))